nixos/modules/system/boot/networkd.nix

```diff
@@ -1969,6 +1950,32 @@ in
         };
       };
 
+    })
+  ];
+
+  stage2Config = mkMerge [
```

Comment:
What is stage 2 here? I assumed stage 2 was this other bash script that we have, that invokes the real system (and we might be able to absorb everything into a systemd-powered stage 1…)

Reply:
Stage 2 just refers to the whole OS after we leave initrd, as I understand the term.